lib/components/home/artwork_rail.js

```diff
@@ -96,9 +101,12 @@ class ArtworkRail extends React.Component {
 
 const styles = StyleSheet.create({
   title: {
-    margin: 10,
-    fontSize: 20,
+    margin: isPad ? 40 : 20,
```

Comment:
at some point we should think about whether it makes sense to make `ipad_title` and `iphone_title` styles instead of doing this little check a bunch of times.